src/containers/forms/PractitionerForm/UserIdSelect/index.tsx

```diff
@@ -0,0 +1,89 @@
+/** wrapper around react-select component that enables one
+ * to select a single user from openmrs
+ */
+import React, { useEffect, useState } from 'react';
+import Select from 'react-select';
+import { ValueType } from 'react-select/src/types';
+import { OpenSRPService } from '../../../../services/opensrp';
+
+const OPENMRS_USERS_REQUEST_PAGE_SIZE = 100;
```

Comment:
Can we please move this to an env variable, with a default int value?  So that we don't have to make a "code change" in the event that we want to change it.